lib/task.js

```diff
@@ -967,4 +975,15 @@ export class Task {
     }
     return success;
   }
+
+  /**
+  Create a new queue using temp credentials.
+
+  @param {credentials} Temporary credentials.
+
+  @return New queue.
+  */
+  createQueue(credentials) {
```

Comment:
Perhaps the status stuff is just added back here.  I'm not sure if we need them, or if we don't. Safer to leave them and sort it out later.